src/System.Device.Gpio/System/Device/Gpio/GpioController.Linux.cs

```diff
@@ -24,11 +24,23 @@ public GpioController(PinNumberingScheme numberingScheme)
         {
         }
 
+        private static GpioDriver GetBestDriverForBoard()
```

Comment:
We should just move the constructor as well as these methods back to GpioController.cs. No need to have them split by filename now that we will only have one build.

Reply:
True. I'll clean that up.